content/en/tracing/trace_collection/admission_controller.md

```diff
@@ -7,12 +7,17 @@ is_beta: true
 
 {{< beta-callout url="#" btn_hidden="true">}}
   Tracing library injection using Admission Controller is in beta. 
-{{< /beta-callout >}} 
+{{< /beta-callout >}}
 
-For your Kubernetes applications that send traces to Datadog by the Cluster Agent (version 7.39 and higher), you can configure the Admission Controller to inject APM Java and JavaScript tracing libraries automatically. By automating the injection of tracer libraries through the admission controller, you don't have to change your application images, helping you get up and running with APM quickly.
+In most environments, configuring your application to send traces to Datadog involves two steps:
```

Comment:
We can be specific in there we are talking about Kubernetes environment so the steps should be clear